qutebrowser/qutebrowser.py

```diff
@@ -112,26 +112,10 @@ def get_argparser():
                        "temporary basedir.")
     debug.add_argument('--no-err-windows', action='store_true', help="Don't "
                        "show any error windows (used for tests/smoke.py).")
-    # For the Qt args, we use store_const with const=True rather than
-    # store_true because we want the default to be None, to make
-    # utils.qt:get_args easier.
-    debug.add_argument('--qt-name', help="Set the window name.",
-                       metavar='NAME')
-    debug.add_argument('--qt-style', help="Set the Qt GUI style to use.",
-                       metavar='STYLE')
-    debug.add_argument('--qt-stylesheet', help="Override the Qt application "
-                       "stylesheet.", metavar='STYLESHEET')
-    debug.add_argument('--qt-widgetcount', help="Print debug message at the "
-                       "end about number of widgets left undestroyed and "
-                       "maximum number of widgets existed at the same time.",
-                       action='store_const', const=True)
-    debug.add_argument('--qt-reverse', help="Set the application's layout "
-                       "direction to right-to-left.", action='store_const',
-                       const=True)
-    debug.add_argument('--qt-qmljsdebugger', help="Activate the QML/JS "
-                       "debugger with a specified port. 'block' is optional "
-                       "and will make the application wait until a debugger "
-                       "connects to it.", metavar='port:PORT[,block]')
+    debug.add_argument('--qt-arg', help="Pass an argument with a value to Qt.",
+                       nargs=2)
+    debug.add_argument('--qt-flag', help="Pass a argument to Qt as flag.",
```

Comment:
As @Kingdread mentioned, I was wrong and this should be "an argument" to be proper English :wink: